product_docs/docs/pgd/5/rel_notes/pgd_5.2.0_rel_notes.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Release notes for EDB Postgres Distributed version 5.2.0"
+navTitle: "Version 5.2.0"
+---
+
+EDB Postgres Distributed version 5.2.0 is a minor version of EDB Postgres Distributed.
+This version addresses security vulnerabilities in dependencies for PGD Proxy and PGD CLI.
```

Comment:
This line should be removed. It looks like Sunny initially created this PR by copying file from [v5.1.0](https://github.com/EnterpriseDB/docs/blob/develop/product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx) which also ideally should not have this line. It was only meant for [v5.0.1](https://github.com/EnterpriseDB/docs/blob/develop/product_docs/docs/pgd/5/rel_notes/pgd_5.0.1_rel_notes.mdx).

Reply:
Hi @djw-m, jfyi, I've removed the above line from v5.2.0 and [v5.1.0](https://github.com/EnterpriseDB/docs/blob/develop/product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx) release notes.

Reply:
ah ok, thanks...actually, just saw Lenz's message in slack